frontend/amundsen_application/static/js/pages/LineagePage/styles.scss

```diff
@@ -0,0 +1,49 @@
+// Copyright Contributors to the Amundsen project.
+// SPDX-License-Identifier: Apache-2.0
+
+@import 'variables';
+@import 'typography';
+@import 'layouts';
+
+$pulse-loader-height: 180px;
+
+.lineage-page {
+  .header-title-text {
+    @extend %text-headline-w2;
+  }
+
+  .header-links {
+    .avatar-label {
+      &:focus,
+      &:hover {
+        color: $link-hover-color;
+      }
+    }
+  }
+
+  .lineage-graph-label {
+    margin-left: $spacer-2;
+  }
+
+  .lineage-graph {
+    display: flex;
+    height: calc(
+      100vh - #{$nav-bar-height + $footer-height + $resource-header-height}
+    );
+    margin: auto;
+  }
+}
+
+.pulse-lineage {
+  height: $pulse-loader-height;
+  width: 100%;
+  text-align: center;
+  margin: 15% auto;
+}
+
+.pulse-lineage-media {
+  height: $pulse-loader-height - $spacer-1;
+  background-image: url('../../../images/icons/Lineage.svg');
```

Comment:
Would it be possible to make Lineage.svg an extra SVG react component and use it so, without setting it as a background image?

Reply:
Let's not.
Both animations (shimmer + pulse) are currently background focused. I would rather keep it this way as it is more consistent.

Reply:
Ack. Sounds good.